integration/record.sh

```diff
@@ -0,0 +1,336 @@
+#!/bin/env bash
```

Comment:
any reason you chose the integration directory instead of keeping the scripts under `hack/`?

Reply:
I was thinking ahead of time and trying to eventually turn this into some sort of integration test. I'll rename it.